### PR TITLE
Localization: Add "has been added as a starter!" localization

### PR DESCRIPTION
**Pull request by josericardo-fo**
## What are the changes?

Localizing and translations

## Capturing (en)
image

## Egg Hatching Phase (en)
image

## Capturing (pt_BR)
image

## Egg Hatching Phase (pt_BR)
image

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/es/battle.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const battle: SimpleTranslationEntries = {
   "trainerDefeated": "¡Has derrotado a\n{{trainerName}}!",
   "moneyWon": "¡Has ganado\n₽{{moneyAmount}} por vencer!",
   "pokemonCaught": "¡{{pokemonName}} atrapado!",
+  "addedAsAStarter": "{{pokemonName}} has been\nadded as a starter!",
```
Comment (Jancojicanoder) [inline src/locales/es/battle.ts:18]:
```suggestion
  "addedAsAStarter": "{{pokemonName}} ha sido añadido\ncomo un inicial!",
```

**Comment by Jancojicanoder** [comment]
Sorry, I changed the suggested spanish if you can check it again

while "ha sido añadido/ncomo un inicial!" is consistent with the portuguese translation and english version of pokerogue
"se ha añadido/na tus iniciales!" is more in line with how the main line games add pokemon to your party or pokedex

both are correct

**Review by EnochG1 — APPROVED** [review]

File: src/locales/de/battle.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ export const battle: SimpleTranslationEntries = {
   "trainerDefeated": "{{trainerName}}\nwurde besiegt!",
   "moneyWon": "Du gewinnst\n{{moneyAmount}} ₽!",
   "pokemonCaught": "{{pokemonName}} wurde gefangen!",
+  "addedAsAStarter": "{{pokemonName}} has been\nadded as a starter!",
```
Comment (CodeTappert) [inline src/locales/de/battle.ts:18]:
```suggestion
  "addedAsAStarter": "{{pokemonName}} wurde als Starterpokémon hinzugefügt!",
```

File: src/locales/fr/battle.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ export const battle: SimpleTranslationEntries = {
   "trainerDefeated": "Vous avez battu\n{{trainerName}} !",
   "moneyWon": "Vous remportez\n{{moneyAmount}} ₽ !",
   "pokemonCaught": "Vous avez attrapé {{pokemonName}} !",
+  "addedAsAStarter": "{{pokemonName}} has been\nadded as a starter!",
```
Comment (Adri1) [inline src/locales/fr/battle.ts:18]:
```suggestion
  "addedAsAStarter": "{{pokemonName}} est ajouté\ncomme starter !",
```

**Review by Adri1 — COMMENTED** [review]
Added French

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**